torch/_inductor/codegen/triton.py

```diff
@@ -2720,6 +2720,8 @@ def bucketize(
             dtype=indexing_dtype,  # type: ignore[attr-defined]
         )
 
+        result.mask_vars = values.mask_vars  # type: ignore[attr-defined]
```

Comment:
Would we need to add the boundaries mask as well ?

Reply:
added all CSEVariables